src/routes/projects.rs

```diff
@@ -176,7 +188,7 @@ pub fn convert_project(
 }
 
 lazy_static! {
-    static ref RE_URL_SAFE: Regex = Regex::new(r"^[a-zA-Z0-9_-]*$").unwrap();
+    static ref RE_URL_SAFE: Regex = Regex::new(r"\S").unwrap();
```

Comment:
This checks for non space characters in string, not the alpha-numeric-ness of the string. Please revert this change.